builtin/game/item.lua

```diff
@@ -205,6 +207,25 @@ function core.get_node_drops(nodename, toolname)
 	return got_items
 end
 
+local non_player_name = ""
+local function user_name(user)
+	return user and user:is_player() and user:get_player_name() or ""
```

Comment:
* `get_player_name()`: returns `""` if is not a player
From `lua_api.txt`

This expression can be simplified: `return user and user:get_player_name() or ""`